tests/toolchain/gcc_compilation.pm

```diff
@@ -27,6 +28,8 @@ sub run {
     # poo#33376: added to investigate OOM
     assert_script_run 'free -m';
     save_screenshot;
+    sleep 3;
```

Comment:
Why?

Reply:
I cannot find the occurrence atm, but I have seen that rarely screenshotting takes a little more time resulting in an inconsistent screenshot. Thus, there's a chance that openqa would save an image of an already/partially cleared screen, which is useless. But at second glance and thought, we might even get rid of L#30 and L#31. 
If the content of the ``free`` command's output has any value for us, we can just simply redirect it to the serial console. wdyt ?

Reply:
The referenced poo# seems resolved and old, I'm in favor of just dropping that.

Reply:
Make sense, even that particular was dropped from the ``gcc_fortran_compilation.pm`` module.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4709/files